### ISSUE TITLE
ground fill bug: certain svg polygons appear at many times their actual size

### ISSUE THREAD

**Issue by davidperrenoud** [fritzing/fritzing-app#1550]
_From [irasc...@gmail.com](https://code.google.com/u/104729248032245122687/) on July 04, 2011 08:36:04_

Open the stepper motor example and do a ground fill. Underneath the h-bridge appear six rectangular copper-fill polygons which incorrectly draw over wires and other connectors. The svg for most of these polygons is:

<svg xmlns=' http://www.w3.org/2000/svg' width='0.05in' height='0.01in' viewBox='0 0 50 10' >
<g id='groundplane'>
<g id='connector0pad'>
<polygon fill='#F28A00'  points='
5,0 5,5 0,5 0,10 45,10 45,5 50,5 50,0 
'/>
</g></g>
</svg>

In other words, they are very small rectangles. When the sketch is exported to svg or gerber, the rectangles are correctly sized, so it appears to be a drawing problem.

_Original issue: http://code.google.com/p/fritzing/issues/detail?id=1550_
